origins_translations/js/origins-translations.js

```diff
@@ -92,7 +92,7 @@
 
     $(elm).find('a').each(function () {
       let $link = new URL( $(this).attr('href') );
-      $link.searchParams.set('u', encodeURIComponent(pageUrl));
+      $link.searchParams.set('u', pageUrl);
```

Comment:
Can we trust the pageUrl is already encoded here? See https://stackoverflow.com/questions/2295223/how-to-find-out-if-string-has-already-been-url-encoded/2295286#2295286.  It might be sensible to decode the pageUrl and compare to original pageUrl - if they are identical, the original was not encoded and so should be encoded.  🤷🏻

Reply:
It looks like URL.searchParams.set encodes the string passed to it - so all good ☝🏻